Python/PlenoInterview/Problem1/order.py

```diff
@@ -0,0 +1,40 @@
+class order:
+
+    def __init__(self, numPeople):
+        self.orderItems = []
+        self.numPeople = numPeople
+
+    # Functions of class
+    def getOrderTotal(self):
+        total = 0
+        for order in self.orderItems:
+            total += order.getTotalPrice()
+        return total
+
+    def getTotalPerPerson(self):
+        return self.getOrderTotal()/self.numPeople
+
+    def getOrderItemByName(self, dishName):
+        for order in self.orderItems:
+            if (order.dishName == dishName):
+                return order
+        return None
+
+    def addNewOrderItem(self, orderItem):
+        self.orderItems.append(orderItem)
+
+    def addExistingOrderItem(self, orderItem):
+        oi = self.getOrderItemByName(orderItem.dishName)
+        oi.amount += 1
+
+    def getTotalPrice(self, orderItem):
```

Comment:
This name could be confusing, it would be better to reference it is a `orderItem` total on the name. A suggestion would be `getTotalOrderItemPrice`.

Also, this function seems redundent with `getTotalPrice` method from `orderItem`